main.go

```diff
@@ -1030,6 +1046,10 @@ func init() {
 		command.PersistentFlags().Bool("json", false, "Prints the information in json format")
 	}
 
+	for _, command := range []*cobra.Command{removeCmd} {
+		command.PersistentFlags().Bool("suppress", false, "Suppress confirm")
```

Comment:
Maybe extend the description a little e.g. `Suppress all confirmation dialogues`